src/calc.ts

```diff
@@ -357,3 +357,21 @@ export function calcSavingsTimer(params: ConfiguratorParameters): number {
   const savings = baseCost * savingsCoefficient
   return savings
 }
+
+export function calcSavingsColdShower(params: ConfiguratorParameters): number {
+  const baseCost = calcEffectiveWarmWaterCost(params)
+  const averageShowerTime = 8
+  let savingsCoefficient
+  switch (params.bausubstanz) {
+    case Bausubstanz.Altbau:
```

Comment:
The idea is that if you shower cold, you save 100% of the base cost --> savings coefficient = 1.

But you are right - this can be done better and I updated the code